docs/source/attributes.rst

```diff
@@ -430,6 +430,13 @@ appserver
     these approaches varies between appservers.  See their documentation for more
     details.
 
+- ``app['appserver']['port']``
+
+  - **Default:** None
+  - Bind the appserver to a port on 127.0.0.1.  This is
```

Comment:
Shouldn’t be `0.0.0.0` here?

Reply:
good catch. had to look up the difference.